src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4010,8 +4021,6 @@ public override BoundNode VisitAsOperator(BoundAsOperator node)
                             var operandType = node.Operand.Type;
                             if (operandType?.IsValueType == true)
                             {
-                                // PROTOTYPE(NullableReferenceTypes): Should we worry about a pathological case of boxing nullable value known to be not null?
```

Comment:
Perhaps leave a comment that we decided to ignore this case, and can revisit if it becomes a problem?